validation/src/test/scala/hmda/validation/rules/lar/validity/V265Spec.scala

```diff
@@ -0,0 +1,46 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.LarEditCheckSpec
+import org.scalacheck.Gen
+
+class V265Spec extends LarEditCheckSpec with BadValueUtils {
+  property("All applications must pass") {
+    forAll(larGen) { lar =>
+      lar.mustPass
+    }
+  }
+
+  val invalidDate: Gen[Int] = intOutsideRange(10000101, 99991231)
+
+  property("An application with an invalid date must fail") {
+    forAll(larGen, invalidDate) { (lar, date) =>
+      val badLar = lar.copy(actionTakenDate = date)
+      badLar.mustFail
+    }
+  }
+
+  property("An application missing part of the date must fail") {
```

Comment:
I don't think all of our tests have to be property based as a hard rule (see V280, V285, V290, V295 tests).  In this case, these extra tests aren't actually necessary (the first test will try `200001`, `99991301` and `99991232`).  I included the extra cases simply as clearer examples of an invalid input.